<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>list words</string>
  </property>
  <widget class="QLabel" name="englishLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>English:</string>
   </property>
  </widget>
  <widget class="QLabel" name="uzbekLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Uzbek:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="englishList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>221</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="uzbekList">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>221</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>660</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addNewWordBtn">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>660</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add new word</string>
   </property>
  </widget>
  <widget class="QPushButton" name="menuBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
